lib/webauthn/configuration.rb

```diff
@@ -23,10 +23,12 @@ def self.if_pss_supported(algorithm)
     attr_accessor :rp_id
     attr_accessor :rp_name
     attr_accessor :verify_attestation_statement
+    attr_accessor :credential_options_timeout
 
     def initialize
       @algorithms = DEFAULT_ALGORITHMS.dup
       @verify_attestation_statement = true
+      @credential_options_timeout = 120000
```

Comment:
For the record, there's been this addition to the spec https://github.com/w3c/webauthn/pull/1317/files.